thunder/executors/nvfuserex_impl.py

```diff
@@ -480,18 +481,18 @@ def __call__(self, *args):
         if hasattr(fd, "_selected_device"):
             kwargs["device"] = fd._selected_device
 
-        if nvfuser_version() >= LooseVersion("0.2.23"):
-            # nvFuser expects empty list instead of None values.
-            kwargs["_enable_options"] = self.enable_options if self.enable_options is not None else []
-            kwargs["_disable_options"] = self.disable_options if self.disable_options is not None else []
-
+        if NVFUSER_SUPPORTS_OPTIONS:
```

Comment:
If we really want to optimize this Python, the __call__ method can be generated differently depending on this option. Not performing the LooseVersion construction and the comparison in the function is a great improvement

Reply:
We can just drop the check and require nvFuser 0.2.23+

Reply:
nvFuser 0.2.23+ is the new requirement added in https://github.com/Lightning-AI/lightning-thunder/pull/1840/commits/f11214adb9e77d4d8e5d0aa3f055cb9e92816179.